CMakeLists.txt

```diff
@@ -1289,6 +1296,141 @@ if(MBGL_WITH_METAL)
     )
 endif()
 
+if(MLN_WITH_VULKAN)
+    message(STATUS "Configuring Vulkan renderer backend")
+
+    target_compile_definitions(
+        mbgl-core
+        PRIVATE
+            MLN_RENDER_BACKEND_VULKAN=1
+    )
+
+    target_include_directories(
```

Comment:
We should add an include directory in `vendor`.

Instead, we should create `vendor/vulkan.cmake` and add the needed target(s) there. This also makes sure we don't get any warnings (turned into errors) that should only apply to our code.